qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/versions.tf

```diff
@@ -12,10 +13,12 @@ terraform {
 {%- elif cookiecutter.provider == "gcp" %}
     google = {
       source = "hashicorp/google"
+      version = "=3.89.0"
```

Comment:
is the `=` intentional here? (same for digitalocean as well)

Reply:
Yeah, just a style preference.  Without the = sign works too.  https://www.terraform.io/docs/language/expressions/version-constraints.html